tests/test_commands.py

```diff
@@ -166,6 +166,20 @@ def teardown():
         assert r.acl_setuser(username, enabled=True,
                              remove_passwords=['pass2'])
         assert len(r.acl_getuser(username)['passwords']) == 1
+        # Resets and tests that hashed values are set properly.
+        hashed_password = ('5e884898da28047151d0e56f8dc629'
+                           '2773603d0d6aabbdd62a11ef721d1542d8')
+        assert r.acl_setuser(username, enabled=True, reset=True,
+                             add_hashes=[hashed_password])
+        assert acl['passwords'] == [hashed_password]
```

Comment:
Did you run these tests locally and not skip them? I can't imagine this line passing as the `acl` dict hasn't been refreshed since the `acl_setuser()` call above it. I think you need an `acl_getuser()` here.

FWIW Travis isn't running these tests because it doesn't have Redis Unstable.

Reply:
I did run these locally but not with a few minor modifications for the username.

acl = r.acl_getuser(username) so its calling it with the variable in this.